django/contrib/gis/db/models/functions.py

```diff
@@ -16,12 +16,9 @@
 
 class GeoFuncMixin:
     function = None
-    output_field_class = None
     geom_param_pos = (0,)
 
     def __init__(self, *expressions, **extra):
```

Comment:
This could be removed now.

Reply:
The `__init__()` method? There are ~15 lines below `super()` that seem to be used.

Reply:
Oops, didn't saw it.